src/checkout-node-query.js

```diff
@@ -0,0 +1,41 @@
+import createGid from './create-gid';
+import shippingRateQuery from './shipping-rate-query';
+import lineItemConnectionQuery from './line-item-connection-query';
+import mailingAddressQuery from './mailing-address-query';
+import customAttributeQuery from './custom-attribute-query';
+import orderQuery from './order-query';
+import addFields from './add-fields';
+
+const defaultFields = [
+  'id',
+  'ready',
+  ['lineItems', lineItemConnectionQuery()],
+  ['shippingAddress', mailingAddressQuery()],
+  ['shippingLine', shippingRateQuery()],
+  'requiresShipping',
+  ['customAttributes', customAttributeQuery()],
+  'note',
+  'paymentDue',
+  'webUrl',
+  ['order', orderQuery()],
+  'orderStatusUrl',
+  'taxExempt',
+  'taxesIncluded',
+  'currencyCode',
+  'totalTax',
+  'subtotalPrice',
+  'totalPrice',
+  'completedAt',
+  'createdAt',
+  'updatedAt'
+];
```

Comment:
we do need ready. That tells us if taxes have settled out, or if they're still processing. Plus shops need this. We do want to be able to render orders, so I would keep those links.